stock_demand_estimate/wizards/stock_demand_estimate_wizard_view.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view"
+                id="view_stock_demand_estimate_sheet_form">
+            <field name="name">stock.demand.estimate.sheet.form</field>
+            <field name="model">stock.demand.estimate.sheet</field>
+            <field name="arch" type="xml">
+                <form string="Stock Demand Estimate Sheet">
+                    <group>
+                        <group name="dates">
+                            <label for="date_from" string="Timesheet Period"/>
+                            <div><field name="date_from" class="oe_inline"/> to <field name="date_to" class="oe_inline"/></div>
+                        </group>
+                        <div/>
+                        <group name="attributes">
+                            <field name="location_id"/>
+                        </group>
+                    </group>
+                    <div/>
+                    <group string="Estimated quantity">
```

Comment:
add name attribute to the group. it's not mandatory but more efficient if you need to override the view